compiler/rustc_data_structures/src/hashes.rs

```diff
@@ -57,9 +57,11 @@ impl<D: Decoder> Decodable<D> for Hash64 {
 }
 
 impl StableHasherResult for Hash64 {
+    type Hash = [u64; 2];
```

Comment:
Since we go to the trouble of making this conversion explicit, what do you think about introducing a newtype for the result value:
```rust
pub struct SipHasher128Result(pub [u64; 2]);

impl ExtendedHasher for SipHasher128 {
    type Hash = SipHasher128Result;
}
```
That would provide some clue as to how to interpret those two u64s.

Reply:
Sure, that's a good idea.

Reply:
Done in https://github.com/rust-lang/rustc-stable-hash/pull/8/commits/7676f2d85de5d76d3d3264979ccb2ae12a54598b; part of https://github.com/rust-lang/rustc-stable-hash/pull/8